SimulationAgent/FileWrapper.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved. 
+
+using System.IO;
+
+namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.SimulationAgent
+{
+    public interface IFile
```

Comment:
nite: rename the interface to `IFileSystem`

Reply:
renamed